example-server/app/public/form.js

```diff
@@ -172,6 +172,9 @@ app.controller('SetCustomerController', [
 
     scope.override = function () {
       scope.allowOverride = !scope.allowOverride;
+      if (!scope.allowOverride) {
+        scope.pixelPath = '';
+      }
```

Comment:
What does this do? This is weird to me since calling scope.override() twice will essentially set pixelPath to '', while scope.allowOverride remains in its original value.